src/main/java/com/squareup/kotlinpoet/FileSpec.kt

```diff
@@ -131,6 +135,10 @@ class FileSpec private constructor(builder: FileSpec.Builder) {
     codeWriter.popPackage()
   }
 
+  private fun isMultiLine(comment: CodeBlock): Boolean {
```

Comment:
Wasn't sure whether to pollute `CodeBlock` with this utility method. Does this seem general enough that it could be used somewhere else? We could add it to `CodeBlock` in that case.

Reply:
It's fine having it here for now, we'll have an option of moving it to `CodeBlock` when we have more than one usage.

nit: It would read better if it was an extension function.

Reply:
Cool! 
Re: extension function. It is my understanding that since we already own `CodeBlock`, adding an extension function would be equivalent to adding the function in `CodeBlock` class itself, since for all purposes, it becomes part of `CodeBlock`'s API. Is there any advantage to adding it as extension function?

Reply:
It'll be an extension function private to `FileSpec`, similar to creating a `private static isMultiline(CodeBlock comment)`. Readability in itself is a big advantage IMO.

Reply:
Ahh interesting! Yep, making it private to `FileSpec` makes a lot of sense. Thanks!